cql/fhir-cql/src/main/java/com/ibm/fhir/cql/engine/retrieve/SearchParameterFHIRRetrieveProvider.java

```diff
@@ -72,18 +73,24 @@ public int getMaxCodesPerQuery() {
         return this.maxCodesPerQuery;
     }
 
-    protected abstract Iterable<Object> executeQueries(String dataType, List<SearchParameterMap> queries) throws Exception;
+    protected abstract Iterable<Object> executeQueries(String resourceType, List<SearchParameterMap> queries) throws Exception;
 
     @Override
     public Iterable<Object> retrieve(String context, String contextPath, Object contextValue, String dataType,
         String templateId, String codePath, Iterable<Code> codes, String valueSet, String datePath,
         String dateLowPath, String dateHighPath, Interval dateRange) {
 
+        if (!ModelSupport.isResourceType(dataType)) {
```

Comment:
should dataType just be renamed to resourceType?

Reply:
I wasn't sure and so I played it safe here.  As mentioned at https://github.com/IBM/FHIR/pull/2942#discussion_r743889717 I was debating just ripping the dataType -> resourceType name change out of the PR since its not really related to the rest of the changes...I was just some local edits I had for my own sanity.